lib/src/io_file.dart

```diff
@@ -241,6 +321,8 @@ class _FromXFile implements IOFile {
 
   final XFile _file;
 
+  final int _readStreamChunkSize = 256 * 1024;
```

Comment:
as we use `256 * 1024` twice, can we use a constant for it?

Reply:
Good catch!